src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  user: User | null;
```

Comment:
To fix this issue you should also update type
![2023-09-27_15h53_46](https://github.com/mate-academy/react_add-todo-form/assets/61904995/2f5bb94f-3603-402c-bbde-b26786491fec)

```suggestion
  user?: User;
```